src/SeisSol.h

```diff
@@ -143,6 +144,14 @@ class SeisSol {
    * Get the flop counter
    */
   monitoring::FlopCounter& flopCounter() { return m_flopCounter; }
+  /**
+   * Reference for timeMirrorManagers to be accessed externally when required
+   */
+  std::pair<seissol::ITM::InstantaneousTimeMirrorManager,
```

Comment:
Will there only be two time mirror managers at a time? (pun _not_intended, but welcome anyways)
Or can there be even more than that? (e.g. having multiple time reversals, maybe ping-ponging each other or similar—if that does make sense?)

Reply:
There are two managers, one which scales up and the other which brings the simulation parameters back to normal. In theory we could have more but it makes sense to have them only in even numbers when we want to apply ITM multiple times.

Reply:
Still, if you're not bound to a certain number of it and use them basically always the same way (and they are, as far as I can see), I'd see it as cleaner to have a vector instead of a pair here.

In the `InitModel.cpp` you can explicitly add two managers to the vector then.

Reply:
Well, as we're going to use just two most of the time (I don't see anyone else creating more scenarios of it in the future), I still think a pair is more appropriate than a vector. It would have made more sense to use a vector when we use more or an unknown number of the. 

The output format is not affected by the ITM. ITM just changes the material parameters for a very short period of time.

Reply:
Oh—sorry; I meant input format. I.e. if you have a list of different ITM parameters (that does not work with the current namelists, but we're, by plan, going to replace the input format anyways soon-ish and use YAML iirc instead—there we would have arrays, and could define an array of ITMs if you want to simulate that).

We can keep it as an `std::pair`, although I quite honestly still don't see any advantage here (rather, we always have allocated two dangling ITM managers lying around if the functionality is not used; it may be cleaner to not have them at all in this case). And changing it to an `std::vector` should be done with only a few lines of code. But again, it's ultimately your choice.